scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixFileEvaluation.java

```diff
@@ -0,0 +1,65 @@
+package scalafix.interfaces;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
+public interface ScalafixFileEvaluation {
+
+    /**
+     * @return original path of the file scalafix has evaluated on
+     */
+    Path getEvaluatedPath();
```

Comment:
to highlight that it's a file (and not a directory that can have been passed via `ScalafixArguments#withPaths`)
```suggestion
    Path getEvaluatedFile();
```